packages/leavittbook/src/components/leavitt-person-company-select-demo.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, html, css } from 'lit';
```

Comment:
In leavitt-person-company-select component:
ignore LeavittPersonCompanySelectSelectedEvent and add docs (see LeavittPersonSelect)
add ` *  @fires change - Fired when input is typed in`
add docs for reset and focus methods